btr-web/btr-main-app/components/review-confirm/Certify.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div>
+    <h2 class="font-bold text-lg mt-5" data-cy="certify-section-label">
```

Comment:
In the future, we should pull this stuff actually out of the component (stuff like headers etc ... ), all the stuff that is part of the page. You can leave it for now here, and I will update this when I push my PR. Cause I did similar thing.